### DISCUSSION TITLE
Use BuildKit withRancher Desktop

### DISCUSSION THREAD

**Discussion by ptrstpp950** [rancher-sandbox/rancher-desktop#3083] [Q&A]
I'm trying to build following Dockerfile:
```
FROM alpine:latest
RUN --mount=type=secret,id=pd cat /run/secrets/pd
CMD ls /run/secrets
```
And the result is:
```
Sending build context to Docker daemon  3.072kB
Step 1/3 : FROM alpine:latest
 ---> 9c6f07244728
the --mount option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled
```

I tried to enable BuildKit with env variable, but without a success. Is it possible to build such image in Rancher Desktop?

**Comment by ghost** [comment]
Has there been any update to this question? It looks like others also have it. :)

**Comment by mdscarlos** [comment]
There's a possible fix for it in this discussion https://github.com/rancher-sandbox/rancher-desktop/discussions/1477
Basically, to enable BuildKit, all you have to do is:
1. Run the following commands to enter LIMA VM shell and configure you docker `daemon.json` file:
```
LIMA_HOME="$HOME/Library/Application Support/rancher-desktop/lima" "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl" shell 0
sudo su -
echo '{ "features": { "buildkit": true } }' > /etc/docker/daemon.json
echo 'DOCKER_OPTS="--config-file=/etc/docker/daemon.json"' > /etc/conf.d/docker
```
2. Restart Rancher (Quit and start again).

It worked well for me.
